### PR TITLE
`Assessment`: Show logic for assessment training

### PR DESCRIPTION
**Pull request by Olimbia23**
#### General

- [x] I tested **all** changes and their related features with **all** corresponding user types on a test server.
- [ ] *This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.ase.in.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/development-process.html#naming-conventions-for-github-pull-requests).
#### Server
- [ ] *I followed the [coding and design guidelines](https://docs.artemis.ase.in.tum.de/dev/guidelines/server/).
- [ ] *I added multiple integration tests (Spring) related to the features (with a high test coverage).
- [ ] *I added `@PreAuthorize` and checked the course groups for all new REST Calls (security).
- [ ] *I implemented the changes with a good performance and prevented too many database calls.
- [ ] *I documented the Java code using JavaDoc style.
#### Client
- [x] I followed the [coding and design guidelines](https://docs.artemis.ase.in.tum.de/dev/guidelines/client/).
- [ ] *I added multiple integration tests (Jest) related to the features (with a high test coverage), while following the [test guidelines](https://docs.artemis.ase.in.tum.de/dev/guidelines/client-tests/).
- [ ] *I added `authorities` to all new routes and checked the course groups for displaying navigation elements (links, buttons).
- [ ] *I documented the TypeScript code using JSDoc style.
- [x] I added multiple screenshots/screencasts of my UI changes.
- [x] I translated all newly inserted strings into English and German.
#### Changes affecting Programming Exercises
- [ ] *I tested **all** changes and their related features with **all** corresponding user types on Test Server 1 (Atlassian Suite).
- [ ] *I tested **all** changes and their related features with **all** corresponding user types on Test Server 2 (Jenkins and Gitlab).

[ ] * - not needed

### Motivation and Context

In order to add clarity to the tutors on how exactly their assessment is evaluated, a new section in the instructions view is added. This section should only be shown when tutor is assessing example submissions.
Extra heading is also added in example submissions screen (as shown in the screenshot below)

### Description

I added a new section in the `assessment-instructions.component` as well as the new heading in the `example-text-submissions` and exaple-modeling-submissions` components.

### Steps for Testing

Prerequisites:
- 1 Tutor

1. Log in to Artemis
2. Navigate to Course Management
3. Navigate to Assessment Dashboard of a Text or Modelling exercise that has Example Submissions to **assess** or create example submissions yourself. (you need to be an instructor to create example submissions) 
4. Click on Assess example submission.
5. In the instructions panel on the right you should see the "Assessment Training Evaluation Criteria" section as well as the header "This is an Assessment Training". (see screenshots)

#### Code Review
- [x] Review 1
- [x] Review 2
#### Manual Tests
- [x] Test 1
- [x] Test 2

#### New header indicating that it's an assessment training
Screen Shot 2021-11-27 at 15 53 35

#### New section explaining the assessment logic
logic

### CODE REVIEW

**Review by Malyuk-A — CHANGES_REQUESTED** [review]
Tested on TS1. It is showed just as described 👍
I have found some minimal translation issues and a few nitpicks in the code

File: src/main/webapp/app/exercises/text/manage/example-text-submission/example-text-submission.component.html [hunk 1]
```diff
@@ -72,6 +72,8 @@ <h2>
 
 <jhi-alert></jhi-alert>
 
+<h5 *ngIf="toComplete" class="mt-3" style="font-weight: 300">{{ 'artemisApp.exampleSubmission.home.assessmentTraining' | artemisTranslate }}</h5>
```
[the remark this answers is not in the document]
Comment (Olimbia23) [reply src/main/webapp/app/exercises/text/manage/example-text-submission/example-text-submission.component.html]: You are right. Applied the `fw-light` class.

File: src/main/webapp/i18n/en/assessmentInstructions.json [hunk 2]
```diff
@@ -14,6 +14,10 @@
         "problemStatement": "Problem Statement",
         "sampleSolution": "Example Solution",
         "assessmentInstructions": "Assessment Instructions",
-        "usageCount": "Usage Count"
+        "usageCount": "Usage Count",
+        "trainingEvaluationCriteria": {
+            "title": "Assessment Training Evaluation Criteria",
+            "description": "In order to successfully complete the training, your assessment should match all of the following criteria:<br>&nbsp;1. Number of feedback that you provide should be the same as the instructors.<br>&nbsp;2. Every feedback should have the same usage of assessment instruction and score.<br>&nbsp;3. For feedback with negative score, a reason should be provided."
```
[the remark this answers is not in the document]
Comment (Olimbia23) [reply src/main/webapp/i18n/en/assessmentInstructions.json]: Couldn't commit your translation suggestions as `feedback` is the terminology we agreed on using in Artemis for plural. But I have incorporated the rest.

File: src/main/webapp/i18n/de/assessmentInstructions.json [hunk 3]
```diff
@@ -14,6 +14,10 @@
         "problemStatement": "Aufgabenstellung",
         "sampleSolution": "Beispiellösung",
         "assessmentInstructions": "Bewertungsanweisungen",
-        "usageCount": "Anzahl Anwendungen"
+        "usageCount": "Anzahl Anwendungen",
+        "trainingEvaluationCriteria": {
+            "title": "Bewertungskriterien für das Assessment-Training",
+            "description": "Um das Assessment-Training erfolgreich abzuschließen, sollte Ihre Bewertung alle folgenden Kriterien erfüllen:<br>&nbsp;1. Die Anzahl der Feedbacks, die Sie geben, sollte der der Dozenten entsprechen.<br>&nbsp;2. Jedes Feedback sollte dieselbe Bewertungsanweisung und dieselbe Punktzahl aufweisen.<br>&nbsp;3. Für Feedback mit negativer Punktzahl sollte der Grund angegeben werden."
```
[the remark this answers is not in the document]
Comment (Olimbia23) [reply src/main/webapp/i18n/de/assessmentInstructions.json]: Adjusted the german translation.

**Review by Olimbia23 — COMMENTED** [review]
Thanks a lot for testing and reviewing it @Malyuk-A 
I applied your suggestions 😀

**Review by MerlinMehmed — DISMISSED** [review]
Tested on TS1, works as expected. Code changes also look good.

**Review by filip-gregurevic — APPROVED** [review]
Code changes look good to me

**Review by filip-gregurevic — APPROVED** [review]
Code changes look good to me

**Review by argertboja — DISMISSED** [review]
Tested on TS1 and works as expected 😃 

Well done @Olimbia23 👍🏼

**Review by Malyuk-A — APPROVED** [review]
Thank you for addressing my change requests. Approved 👍

**Pull request merged**